index.js

```diff
@@ -268,6 +300,7 @@ function LandroidLogger(log){
   }
   this.trace = this.noLogMsg;
   this.debug = this.noLogMsg;
+  this.mowdata = this.noLogMsg;
```

Comment:
Is this needed?

Reply:
No idea what it does! I just copied it from the debug switch that was there previously. If I remove it should I be able to test any difference?

Reply:
These just define function names for the underlying library to log into. Its basically a "fake" logging object for the worx library. So yeah, you can remove this - without any visible changes.